tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+
+## Setup Target Database (Cloud Spanner)
+1. Create a cloud spanner instance
+
+        gcloud spanner instances create spanner-tgt \
+            --config=regional-us-central1 \
+            --nodes=1 \
+            --description=spanner-tgt
+
+2. Create database and empty table.
+
+        gcloud spanner databases create employeedb \
+        --instance=spanner-tgt \
+        --ddl='CREATE TABLE signin_log (id STRING(36), employee_email STRING(200), details STRING(500)) PRIMARY KEY (id)'
+        
+**NOTE** You might have noticed that employee_id has been changed to STRING in the above example. As discussed previously, this is done so that application can generate UUID keys post migration.
+
+## Setup Middleware (Striim)
+
+### Deploy Striim
+
+1. Create striim instance from market-place. Visit [this link](https://console.cloud.google.com/marketplace/product/striim/striim-spanner) from gcp market-place to get striim installed.
+
+![Striim Marketplace](1_striim_market_place.png)
+
+2. Click launch and then on the next screen keep everything as defaults and click deploy.
+
+![Striim Deploy Screen](2_striim_deployment.png)
+
+3. Once deployed you will get a link to striim instance along with userid and password.  
+   *NOTE: Using Deployment manager page you can get access to same information again if you forgot your credentials.*
+
+![Striim Credentials](3_striim_deployment_manager.png)
+
+### Configure Striim
+
+1. To allow Striim to communicate with Cloud SQL for MySQL, [add the Striim server's IP address](https://cloud.google.com/sql/docs/mysql/connect-external-app)
+ to the Cloud SQL for MySQL instance's authorized networks. 
+ 
+        STRIIMVM_ZONE=us-central1-f
+        gcloud sql instances patch mysql-57 \
+            --authorized-networks=$(gcloud compute instances describe striim-spanner-1-vm \
+            --format='get(networkInterfaces[0].accessConfigs[0].natIP)' \
+            --zone=$STRIIMVM_ZONE)
+
+Update STRIIMVM_ZONE with the right value.
+
+2. Deploy mysql driver as described [here](https://cloud.google.com/architecture/partners/continuous-data-replication-cloud-spanner-striim#setting_up_mysql_connector_j)
+
+    SSH into Striim VM
+    
+        gcloud compute ssh striim-spanner-1-vm
+
+    Execute below comannds to download and deploy mysql driver inside striim.
+    
+        wget https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.49.tar.gz
+        tar -xvzf mysql-connector-java-5.1.49.tar.gz
+        sudo cp ~/mysql-connector-java-5.1.49/mysql-connector-java-5.1.49.jar /opt/striim/lib
+        sudo chmod +x /opt/striim/lib/mysql-connector-java-5.1.49.jar
+        sudo chown striim /opt/striim/lib/mysql-connector-java-5.1.49.jar
+        sudo systemctl stop striim-node
+        sudo systemctl stop striim-dbms
+        sudo systemctl start striim-dbms
+        sudo systemctl start striim-node
+
```

Comment:
Should we remove the commands in  this section and just link to the documentation page ? Saves 
```suggestion
2. Setup MySQL Connector to connect Striim to your Cloud SQL for MySQL instance. Follow the instructions here [here](https://cloud.google.com/architecture/partners/continuous-data-replication-cloud-spanner-striim#setting_up_mysql_connector_j)
```

Reply:
I would prefer to keep the commands.